v21.2/cluster-setup-troubleshooting.md

```diff
@@ -284,30 +284,38 @@ See the following FAQs:
 
 ## Capacity planning issues
 
-Following are some of the possible issues you might have while planning capacity for your cluster:
+You may encounter the following issues when your cluster nears 100% resource capacity:
 
 -   Running CPU at close to 100% utilization with high run queue will result in poor performance.
 -   Running RAM at close to 100% utilization triggers Linux OOM and/or swapping that will result in poor performance or stability issues.
 -   Running storage at 100% capacity causes writes to fail, which in turn can cause various processes to stop.
--   Running storage at 100% utilization read/write will causes poor service time.
+-   Running storage at 100% utilization read/write causes poor service time and [node shutdown](operational-faqs.html#what-happens-when-a-node-runs-out-of-disk-space).
 -   Running network at 100% utilization causes response between databases and client to be poor.
 
 **Solution:** [Access the DB Console](ui-overview.html#db-console-access) and navigate to **Metrics > Hardware** dashboard to monitor the following metrics:
 
-First, check adequate capacity was available for the incident for the following components.
+Check that adequate capacity was available for the incident:
 
 Type | Time Series | What to look for
 --------|--------|--------|
 RAM capacity | Memory Usage | Any non-zero value
 CPU capacity | CPU Percent | Consistent non-zero values
+Disk capacity | Available Disk Capacity | Any non-zero value
+IOPS | Disk IOPS in Progress | Zero or occasional single-digit values
 Network capacity | Network Bytes Received<br/>Network Bytes Sent | Any non-zero value
 
-Check Near Out of Capacity Conditions:
+{{site.data.alerts.callout_info}}
+For minimum provisioning guidelines, see [Basic hardware recommendations](recommended-production-settings.html#basic-hardware-recommendations).
+{{site.data.alerts.end}}
+
+Check for resources that are running out of capacity:
 
 Type | Time Series | What to look for
 --------|--------|--------|
 RAM capacity | Memory Usage | Consistently more than 80%
-CPU capacity | CPU Percent | Consistently less than 20% in idle (i.e.:80% busy)
+CPU capacity | CPU Percent | Consistently less than 20% in idle (i.e., 80% busy)
+Disk capacity | Available Disk Capacity | Consistently less than 20% of the [store](cockroach-start.html#store) size
```

Comment:
As I mentioned above: I am pretty sure this is an arbitrary value I picked up through previous threads - unfortunately can't find the exact source now. Do we have more precise guidance?